pkg/master/master.go

```diff
@@ -200,6 +202,11 @@ func setDefaults(c *Config) {
 		}
 		c.PortalNet = portalNet
 	}
+	if c.PortalServicePorts.Size() == 0 {
+		defaultPortRange := util.PortRange{MinInclusive: 10000, MaxExclusive: 11000}
```

Comment:
How could we NOT do this for clouds that don't require it?  Or is that just not worth the effort?  If NeedsClusterServicePort() is false, we shouldn't need anything here...